src/App.js

```diff
@@ -1,13 +1,59 @@
 import React from 'react';
 
 import './App.scss';
+import { GoodsList } from './components/GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component {
+  state = {
+    goodsList: [],
+  }
+
+  allGoods = async() => {
+    const goods = await getAll();
+
+    this.setState({ goodsList: goods });
+  }
+
+  first5Goods = async() => {
+    const goods = await get5First();
+
+    this.setState({ goodsList: goods });
+  }
+
+  redGoods = async() => {
+    const goods = await getRedGoods();
+
+    this.setState({ goodsList: goods });
+  }
```

Comment:
You can create 1 function with callback parameter and call this function in onClick with correct callback